src/pspm_glm.m

```diff
@@ -244,25 +244,26 @@
 end
 
 %% 3 Check & get data
-fprintf('Getting data ...');
 nFile = numel(model.datafile);
 for iFile = 1:nFile
-  [sts, ~, data] = pspm_load_data(model.datafile{iFile}, model.channel);
-  if sts < 1
-    return
-  end
+  % 3.1 User output
+  fprintf('GLM analysis: %s ...', model.datafile{iFile});
+  % 3.2 Check whether model file exists
```

Comment:
> the commented check makes sense but it should be outside the if-loop. Why is it commented?

Because it should be assigned at line 231 together with other option fields.